src/client/visualizations/scatterplot/point.tsx

```diff
@@ -27,19 +27,33 @@ interface PointProps {
   yScale: LinearScale;
   xSeries: ConcreteSeries;
   ySeries: ConcreteSeries;
+  onHover(datum: Datum): void;
```

Comment:
Maybe `setHover` and `resetHover`? `Point` doesn't need to understand what does it mean that hover is null.

Note: We have very lenient tsconfig, but don't define function with specific type parameter and pass null.

Reply:
Maybe it's too lenient 🤔  Anyway, good point, resolved!

Reply:
It is. But changing it to more strict resulted in one big scary pile of errors 😱